tests/util/setup-smoke-tests.ts

```diff
@@ -39,5 +39,14 @@ export function describeSmokeSuite(
     });
 
     cb(context, testIt);
+
+    afterEach(async function () {
+      // This timeout added to give the Websockets enough time to recover when running on K8 pods
+      if (this.currentTest.state !== "passed") {
+        await new Promise((resolve) => {
```

Comment:
In node16 we can use the new `setTimeout`:

````typescript
import { setTimeout } from "timers/promises";

await setTimeout(1000);
```

Reply:
ah, nice. Done!